packages-exp/auth-exp/src/api/index.ts

```diff
@@ -125,7 +125,8 @@ export async function _performFetchWithErrorHandling<V>(
       return response.json();
     } else {
       const json: JsonError = await response.json();
-      const authError = errorMap[json.error.message];
+      const serverErrorCode = json.error.message.split(' : ')[0] as ServerError;
```

Comment:
consider using a regex to make this potentially-less-brittle
`/\s*:/` should do it

Reply:
looks like the existing SDK does it a bit different... https://github.com/firebase/firebase-js-sdk/blob/master/packages/auth/src/rpchandler.js#L2916 they use everything to the right of the colon as the message, overriding whatever is in the error map